### ISSUE TITLE
Function to tag all as read/unread

### ISSUE THREAD

**Issue by schvabodka-man** [skeeto/elfeed#238]
Hi would be great if there would be interactive functions to tag all your feeds as unread/read

**Comment by skeeto** [comment]
Create a selection (C-SPC, C-x h, etc.), then r/u/+/- will modify every 
entry in the selection in one command. Use a filter (s, S) to narrow the 
displayed entries if necessary.

**Comment by schvabodka-man** [comment]
Yes, i can do that, but it would be a lot better if i can just M-x elfeed-mark-all-read

**Comment by skeeto** [comment]
You can easily add this feature to your own configuration:

    (defun my-elfeed-mark-all-read ()
      (interactive)
      (elfeed-untag elfeed-search-entries 'unread)
      (elfeed-search-update :force)) ; redraw

I don't want to add this to Elfeed because it doesn't need a unique 
function for every little thing someone might want to do. Far better to 
to have composable primitives whose cross product covers those needs — 
such as elfeed-search-untag-all applying to a selection.

**Comment by alphapapa** [comment]
@schvabodka-man You might find this useful: https://github.com/skeeto/elfeed/issues/250

**Comment by danrobi11** [comment]
@skeeto 

> I don't want to add this to Elfeed because it doesn't need a unique
function for every little thing someone might want to do.

I dont think a `Mark-All-As-Read` function is a random "thing someone might want to do" 
Mark all as read is a common option in regular feed reader clients.

Although, thanks for the workaround
